crates/store/re_dataframe/src/query.rs

```diff
@@ -345,13 +345,8 @@ impl<E: StorageEngineLike> QueryHandle<E> {
                         let query =
                             re_chunk::LatestAtQuery::new(TimelineName::new(""), TimeInt::STATIC);
 
-                        let component_descriptor = store
-                            .entity_component_descriptor(&descr.entity_path, descr.component)
-                            .into_iter()
-                            .next()?;
-
                         let results =
-                            cache.latest_at(&query, &descr.entity_path, [&component_descriptor]);
+                            cache.latest_at(&query, &descr.entity_path, [descr.component]);
```

Comment:
The deletion above is haunting me, but I guess that was just an early exit? Looking at the diff, we could have always just passed `descr` along.

Or did that line just become trivial because of @IsseW's recent refactor?

Reply:
I believe recently we had to add a brunch of descriptor lookups because of the chnage to component target, and those are gone now again because the descriptor is available in a more obvious fashion

don't see why it would be needed since we're asking the same store on the latest_at query then for the existance